src/App.vue

```diff
@@ -21,7 +22,7 @@
                 <h3>
                     Calendar in full view
                 </h3>
-                <DatePicker :alwaysVisible="true" :showYear="true" :i18n="i18n" />
+                <DatePicker :alwaysVisible="true" monthFormat="MMMM YYYY" :i18n="i18n" />
```

Comment:
As it's the default, it would be better to remove the monthFormat prop (to avoid confusion, let users understand it's the default format)